lib/resources/r4/condition.yaml

```diff
@@ -100,18 +166,18 @@ fields:
     </ul>
 
```

Comment:
can you remove this extra line?

Reply:
Removed [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/aea29d63a9a4c4f36b0b2272e8ae75b115102dbf).